commerce_coordinator/apps/commercetools/pipeline.py

```diff
@@ -108,24 +109,14 @@ def run_filter(self, active_order_management_system, order_number, **kwargs):  #
             duration = (datetime.now() - start_time).total_seconds()
             log.info(f"[Performance Check] get_order_by_number call took {duration} seconds")
 
+            intent_id, psp = get_edx_payment_info(ct_order)
+
             ret_val = {
                 "order_data": ct_order,
+                "psp": psp,
+                "payment_intent_id": intent_id
             }
 
-            intent_id = get_edx_payment_intent_id(ct_order)
-
-            if intent_id:
-                ct_payment = ct_api_client.get_payment_by_key(intent_id)
-                ret_val['payment_intent_id'] = intent_id
-                ret_val['amount_in_cents'] = get_edx_refund_amount(ct_order)
-                ret_val['has_been_refunded'] = has_refund_transaction(ct_payment)
-                ret_val['payment_data'] = ct_payment
-            else:
-                ret_val['payment_intent_id'] = None
-                ret_val['amount_in_cents'] = decimal.Decimal(0.00)
-                ret_val['has_been_refunded'] = False
-                ret_val['payment_data'] = None
-
```

Comment:
I have removed these extra properties because these are not useable in this pipeline. I have also verified that this pipeline is also not used any other place except view receipt flow..